Mathlib/GroupTheory/Exponent.lean

```diff
@@ -106,8 +106,15 @@ theorem exponentExists_iff_ne_zero : ExponentExists G ↔ exponent G ≠ 0 := by
 #align add_monoid.exponent_exists_iff_ne_zero AddMonoid.exponentExists_iff_ne_zero
 
 @[to_additive]
-theorem exponent_eq_zero_iff : exponent G = 0 ↔ ¬ExponentExists G := by
-  simp only [exponentExists_iff_ne_zero, Classical.not_not]
+alias ⟨ExponentExists.exponent_ne_zero, _⟩ := exponentExists_iff_ne_zero
+
+@[to_additive]
+theorem ExponentExists.exponent_pos (h : ExponentExists G) : 0 < exponent G :=
+  h.exponent_ne_zero.bot_lt
+
+@[to_additive]
+theorem exponent_eq_zero_iff : exponent G = 0 ↔ ¬ExponentExists G :=
+  exponentExists_iff_ne_zero.symm.not_right
```

Comment:
Can you
* turn around `exponentExists_iff_ne_zero`
* write `exponent_pos : 0 < exponent G ↔ ExponentExists G`
* derive `protected alias ⟨_, ExponentExists.exponent_pos⟩ := exponent_pos`